### PR TITLE
Normalize `with open` context handler variable

### PR DESCRIPTION
**Pull request by cidrblock**
There some a slew of variations here, `infile`, `outfile`, `fhand`, `filehand`, most likely because of contributors try to avoid the pylint rule. I find this to be a good example of where a linting rule happened to work against us.

```

************* Module src.ansible_navigator.actions.help_doc
src/ansible_navigator/actions/help_doc.py:37:13: C0103: Variable name "fh" doesn't conform to snake_case naming style (invalid-name)
```

This simply makes them all the same throughout the project and eliminates some non-words and abbreviations.

- `fh` was chosen as it simply appears to be common and possibly a familiar pattern to contributors.
- Also allow `fh` in our pylint configuration

https://sourcegraph.com/search?q=context:global+with+open+.*+as+fh:+lang:python+&patternType=regexp

Although I did not find any documentation, I'm assuming is a common abbreviation of "file handle"

Note: The file object is never used far beyond the context handler in our case.

### CODE REVIEW

**Review by relrod — CHANGES_REQUESTED** [review]
This one is a bit odd to me and probably worth discussing. Are we going to force that people who write new code using context handlers to manipulate/read files also name variables the same way?

If yes: Now we're to the point of dictating variable names which seems overzealous to me.
If no: Then this "inconsistency" is going to get reintroduced sooner than later.

I think in general, I'm not in favor of this change; I don't think what the file handle object is called will have much if any impact on the readability of the code. I often see `f` or `fh` used, and I actually prefer those shorter ones, but I wouldn't try to enforce that style either.

**Comment by cidrblock** [comment]
No desire to ever enforce this.............  It was my spell checker that it annoyed, so I just fixed them all.

I'll work with you to find an alternative.

**Comment by cidrblock** [comment]
Thanks @relrod I tried to incorporate what we talked about in the description.  Good call out about navigator being welcoming to contributors and us consciously needing to make an effort to not turn people off or away.

File: share/ansible_navigator/utils/catalog_collections.py [hunk 1]
```diff
@@ -64,9 +64,9 @@ def _catalog_plugins(self, collection: Dict) -> None:
         if file_manifest_file:
             file_path = f"{path}/{file_manifest_file}"
             if os.path.exists(file_path):
-                with open(file=file_path, encoding="utf-8") as read_file:
+                with open(file=file_path, encoding="utf-8") as fh:
```
Comment (relrod) [inline share/ansible_navigator/utils/catalog_collections.py:67]: Should we drop the `file=` while we're at it?

**Review by relrod — APPROVED** [review]

File: tests/integration/actions/config/base.py [hunk 2]
```diff
@@ -120,8 +120,8 @@ def test(self, request, tmux_session, step):
 
         if not any((step.look_fors, step.look_nots)):
             dir_path, file_name = fixture_path_from_request(request, step.step_index)
-            with open(file=f"{dir_path}/{file_name}", encoding="utf-8") as infile:
-                expected_output = json.load(infile)["output"]
+            with open(file=f"{dir_path}/{file_name}", encoding="utf-8") as fh:
```
[the remark this answers is not in the document]
Comment (cidrblock) [reply tests/integration/actions/config/base.py:123]: pathlib instead probably....

File: share/ansible_navigator/utils/catalog_collections.py (hunk 1 above)
Comment (cidrblock) [reply share/ansible_navigator/utils/catalog_collections.py:67]: I'll get it when I pathlib join them all

File: pyproject.toml [hunk 3]
```diff
@@ -78,6 +78,8 @@ skip_glob = ["tests/fixtures/common/collections*"] # Skip ansible content due to
     [tool.pylint.master]
     # tm_tokenize is virtually vendored and shouldn't be linted as such
     ignore = "tm_tokenize"
+    # pylint defaults + fh for with open .. as fh
+    good-names = "i,j,k,ex,Run,_,fh"
```
[the remark this answers is not in the document]
Comment (cidrblock) [reply pyproject.toml]: on it

**Pull request merged**